config.toml

```diff
@@ -8,3 +8,11 @@ appendable_files = [
 configlet_configurable_keys = [
     "fmt"
 ]
+
+# File-specific config
+[files]
+[files."CODE_OF_CONDUCT.md"]
+trailing_newlines = 2
```

Comment:
This was there just to prove it was working, right? https://github.com/exercism/julia/pull/500/files

Reply:
Kind of. I think it's sensible that any append to that file is separated by two newlines as a new paragraph.